xain_fl/grpc/test_grpc.py

```diff
@@ -146,6 +146,7 @@ def test_participant_heartbeat(mock_heartbeat_request, _mock_sleep, _mock_event)
     mock_heartbeat_request.assert_has_calls([mock.call(), mock.call()])
 
 
+@pytest.mark.skip("Skipping due to moving of the grpc participant as sdk to xain-sdk")
```

Comment:
Can't we just remove these tests or replace them with the updated tests in `xain_fl/grpc/test_grpc_updated.py`?

Reply:
Agree. Removed the old `test_grpc`module and renamed the updated one.